web/app/solar.js

```diff
@@ -30,14 +38,19 @@ exports.draw = function(canvasSelector, now, grib1, grib2, solarColor, projectio
     if (moment(now) > moment(t_after)) {
         return console.error('Error while interpolating solar because current time is out of bounds');
     }
-    var k = (now - t_before)/(t_after - t_before);
+
+
+    var k = (now - t_before) / (t_after - t_before);
     var buckets = d3.range(solarColor.range().length)
-        .map(function(d) { return []; });
+        .map(function (d) { return []; });
     var bucketIndex = d3.scaleLinear()
         .rangeRound(d3.range(buckets.length))
         .domain(solarColor.domain())
         .clamp(true);
-    var k = (now - t_before)/(t_after - t_before);
+    var solarScale = solarColor.domain()[solarColor.domain().length - 1];
+
+    var k = (now - t_before) / (t_after - t_before);
+    if (!k || !isFinite(k)) k = 0;
```

Comment:
ok then we should probably remove that. I'm sure I understand why you have twice the same grid. There should be two grids, one at the beginning of the hour, the other at the end.